angular/core/services/exception_handler.coffee

```diff
@@ -1,11 +1,11 @@
-angular.module('loomioApp').factory '$exceptionHandler', (AppConfig) ->
-  (exception, cause) ->
-    if AppConfig.reportErrors
-      Airbrake.push
-        error:
-          message : exception.toString()
-          stack : exception.stack
-        params:
-          user_id: AppConfig.currentUserId
-
-    console.error "LoomioApp exception:", exception, cause
+# angular.module('loomioApp').factory '$exceptionHandler', (AppConfig) ->
```

Comment:
maybe this is the time when we start to report client exceptions to
airbrake? (maybe as a seperate account?)

On Tue, 1 Mar 2016 at 11:30 James Kiesel <notifications@github.com> wrote:

> In angular/core/services/exception_handler.coffee
> <https://github.com/loomio/loomio/pull/3010#discussion_r54490538>:
>
> > @@ -1,11 +1,11 @@
> > -angular.module('loomioApp').factory '$exceptionHandler', (AppConfig) ->
> > -  (exception, cause) ->
> > -    if AppConfig.reportErrors
> > -      Airbrake.push
> > -        error:
> > -          message : exception.toString()
> > -          stack : exception.stack
> > -        params:
> > -          user_id: AppConfig.currentUserId
> > -
> > -    console.error "LoomioApp exception:", exception, cause
> > +# angular.module('loomioApp').factory '$exceptionHandler', (AppConfig) ->
>
> Have to remember to uncommit this. Actually I might make this so it
> console logs the exception stack, which would be much helpful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3010/files#r54490538>.
>